app/paths.js

```diff
@@ -156,13 +156,18 @@ module.exports = {
     addMetadata: '/create-payment-link/add-reporting-column',
     editMetadata: '/create-payment-link/add-reporting-column/:metadataKey',
     deleteMetadata: '/create-payment-link/add-reporting-column/:metadataKey/delete',
-    manage: '/create-payment-link/manage',
+    managePage: '/create-payment-link/manage',
     disable: '/create-payment-link/manage/disable/:productExternalId',
     delete: '/create-payment-link/manage/delete/:productExternalId',
```

Comment:
should we put the other `/manage/` paths under `manage` too? If just some are under there, it seems a little confusing.